src/python.gen.ts

```diff
@@ -367,6 +366,30 @@ ${this.hooks.join('\n')}
       + `${returnStmt}`
   }
 
+  bodyParamsTypeAssertions(indent: string, bodyParams: IParameter[]): string {
+    const bump = indent + this.indentStr
+    let assertions: string = ''
+    if (bodyParams.length > 0) {
```

Comment:
This is a question for @jkaster (not germane to this particular PR) -

it seems odd that we've implemented the body param as an array (`method.bodyParams`). In the old swagger spec a method just has a parameters list and presumably only one has the `"in": "body"` property. But maybe that could be multiple? However, the openAPI spec has a separate `requestBody` top level key for a method and does not have an `"in": "body"` type `"parameters"` instance:

swagger
```json
    "/content_metadata_access/{content_metadata_access_id}": {
      "put": {
        "tags": [
          "Content"
        ],
        "operationId": "update_content_metadata_access",
        "summary": "Update Content Metadata Access",
        "description": "### Update type of access for content metadata.\n",
        "parameters": [
          {
            "name": "content_metadata_access_id",
            "in": "path",
            "description": "Id of content metadata access",
            "required": true,
            "type": "string"
          },
          {
            "name": "body",
            "in": "body",
            "description": "Content Metadata Access",
            "required": true,
            "schema": {
              "$ref": "#/definitions/ContentMetaGroupUser"
            }
          }
        ],
```

openAPI:
```json
    "/content_metadata_access/{content_metadata_access_id}": {
      "put": {
        "tags": [
          "Content"
        ],
        "operationId": "update_content_metadata_access",
        "summary": "Update Content Metadata Access",
        "description": "### Update type of access for content metadata.\n",
        "parameters": [
          {
            "name": "content_metadata_access_id",
            "in": "path",
            "description": "Id of content metadata access",
            "required": true,
            "schema": {
              "type": "string"
            }
          }
        ],
        "requestBody": {
          "$ref": "#/components/requestBodies/ContentMetaGroupUser"
        },
```

Reply:
"How many body parameters?" is a really good question that mirrors the investigation I had while starting this project. From what I've seen for **our** API we'll only ever have one body parameter, and our generator basically caters to that. It still seems possible we could encounter a spec that has multiple body parameters.